composer.json

```
@@ -4,13 +4,18 @@
     "type": "phpcodesniffer-standard",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4 || 8.0.* || 8.1.*",
+        "php": "^7.4 || 8.0.* || 8.1.* || 8.2.* || 8.3.*",
```

I think we can be more flexible and just allow PHP 7.4 and PHP 8.x.
```suggestion
        "php": "^7.4 || ^8.0",
```